README.md

```diff
@@ -585,6 +586,21 @@ symbol.symbol # A-CV
 
 See [#international-symbols](https://iexcloud.io/docs/api/#international-symbols).
 
+### Get Latest Foreign Exchange Rates
+
+Returns an array of foreign exchange rates for a given list of symbols
+
+```ruby
+rates = client.fx_latest(['USDCAD', 'USDGBP', 'USDJPY'])
+
+rate = rates.first
+rate.symbol # USDCAD
+rate.rate # 1.25674
+rate.timestamp # 1627045829863
```

Comment:
Is this a DateTime?

Reply:
it's just an Integer (Unix timestamp), but maybe it would make sense to parse it as a DateTime?

Reply:
If that doesn't lose information, definitely yes.